recipes/roifile/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "roifile" %}
+{% set version = "2020.2.12" %}
+{% set sha256 = "e7c140117d5d9cd48a0902ac1284a6be09c3bc936d6ff1a1c07118c991abcd50" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  noarch: python
+  entry_points:
+    - roifile = roifile.roifile:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy >=1.14.5
+
+test:
+  imports:
+    - roifile
+  commands:
+    - roifile
+
+about:
+  home: https://github.com/cgohlke/roifile
+  license: BSD
```

Comment:
Which version of BSD is this? This entry should be an SPDX identifier

Reply:
BSD-3-Clause … fixed in next commit.